<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>menu</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/m.jpg</normaloff>images/m.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/图标1.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线 Light</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>校园猫管理平台</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_hello">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>751</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线 Light</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_level">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线 Light</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>190</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/图标5.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/图标6.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>91</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/图标3.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>310</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/图标4.jpg</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_findcat">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>210</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线 Light</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>浏览/查询猫猫信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_feedcat">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>330</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线 Light</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>猫猫投喂登记</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_addcat">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>210</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线 Light</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>新增猫猫</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_seecat">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>330</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线 Light</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>猫猫出现位置打卡</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
